<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" total="4" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="28672" started-at="2013-10-26T12:13:33Z" finished-at="2013-10-26T12:14:02Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="28672" started-at="2013-10-26T12:13:33Z" finished-at="2013-10-26T12:14:02Z">
      <class name="com.example.tests.GroupCreationTests">
        <test-method status="PASS" signature="testEmptyGroupCreation()[pri:0, instance:com.example.tests.GroupCreationTests@3e006649]" name="testEmptyGroupCreation" duration-ms="7601" started-at="2013-10-26T16:13:54Z" finished-at="2013-10-26T16:14:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testEmptyGroupCreation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.example.tests.GroupCreationTests@3e006649]" name="tearDown" is-config="true" duration-ms="320" started-at="2013-10-26T16:14:02Z" finished-at="2013-10-26T16:14:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="FAIL" signature="testNonEmptyGroupCreation()[pri:0, instance:com.example.tests.GroupCreationTests@3e006649]" name="testNonEmptyGroupCreation" duration-ms="2" started-at="2013-10-26T16:14:02Z" finished-at="2013-10-26T16:14:02Z">
          <exception class="org.openqa.selenium.remote.SessionNotFoundException">
            <message>
              <![CDATA[The FirefoxDriver cannot be used after quit() was called.
Build info: version: '2.37.0', revision: 'a7c61cb', time: '2013-10-18 17:14:00'
System info: host: 'RMFIWAS', ip: '192.168.1.102', os.name: 'Windows 8', os.arch: 'amd64', os.version: '6.2', java.version: '1.7.0_45'
Driver info: driver.version: RemoteWebDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.remote.SessionNotFoundException: The FirefoxDriver cannot be used after quit() was called.
Build info: version: '2.37.0', revision: 'a7c61cb', time: '2013-10-18 17:14:00'
System info: host: 'RMFIWAS', ip: '192.168.1.102', os.name: 'Windows 8', os.arch: 'amd64', os.version: '6.2', java.version: '1.7.0_45'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.execute(FirefoxDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:527)
	at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:276)
	at com.example.tests.TestBase.openMainPage(TestBase.java:77)
	at com.example.tests.GroupCreationTests.testNonEmptyGroupCreation(GroupCreationTests.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.remote.SessionNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testNonEmptyGroupCreation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.example.tests.GroupCreationTests@3e006649]" name="tearDown" is-config="true" duration-ms="1" started-at="2013-10-26T16:14:02Z" finished-at="2013-10-26T16:14:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.example.tests.GroupCreationTests -->
      <class name="com.example.tests.ContactCreationTests">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.example.tests.ContactCreationTests@226dcb0f]" name="setUp" is-config="true" duration-ms="20715" started-at="2013-10-26T16:13:33Z" finished-at="2013-10-26T16:13:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testEmptyContactCreation()[pri:0, instance:com.example.tests.ContactCreationTests@226dcb0f]" name="testEmptyContactCreation" duration-ms="1" started-at="2013-10-26T16:14:02Z" finished-at="2013-10-26T16:14:02Z">
          <exception class="org.openqa.selenium.remote.SessionNotFoundException">
            <message>
              <![CDATA[The FirefoxDriver cannot be used after quit() was called.
Build info: version: '2.37.0', revision: 'a7c61cb', time: '2013-10-18 17:14:00'
System info: host: 'RMFIWAS', ip: '192.168.1.102', os.name: 'Windows 8', os.arch: 'amd64', os.version: '6.2', java.version: '1.7.0_45'
Driver info: driver.version: RemoteWebDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.remote.SessionNotFoundException: The FirefoxDriver cannot be used after quit() was called.
Build info: version: '2.37.0', revision: 'a7c61cb', time: '2013-10-18 17:14:00'
System info: host: 'RMFIWAS', ip: '192.168.1.102', os.name: 'Windows 8', os.arch: 'amd64', os.version: '6.2', java.version: '1.7.0_45'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.execute(FirefoxDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:527)
	at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:276)
	at com.example.tests.TestBase.openMainPage(TestBase.java:77)
	at com.example.tests.ContactCreationTests.testEmptyContactCreation(ContactCreationTests.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.remote.SessionNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testEmptyContactCreation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.example.tests.ContactCreationTests@226dcb0f]" name="tearDown" is-config="true" duration-ms="1" started-at="2013-10-26T16:14:02Z" finished-at="2013-10-26T16:14:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="FAIL" signature="testNonEmptyContactCreation()[pri:0, instance:com.example.tests.ContactCreationTests@226dcb0f]" name="testNonEmptyContactCreation" duration-ms="1" started-at="2013-10-26T16:14:02Z" finished-at="2013-10-26T16:14:02Z">
          <exception class="org.openqa.selenium.remote.SessionNotFoundException">
            <message>
              <![CDATA[The FirefoxDriver cannot be used after quit() was called.
Build info: version: '2.37.0', revision: 'a7c61cb', time: '2013-10-18 17:14:00'
System info: host: 'RMFIWAS', ip: '192.168.1.102', os.name: 'Windows 8', os.arch: 'amd64', os.version: '6.2', java.version: '1.7.0_45'
Driver info: driver.version: RemoteWebDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.remote.SessionNotFoundException: The FirefoxDriver cannot be used after quit() was called.
Build info: version: '2.37.0', revision: 'a7c61cb', time: '2013-10-18 17:14:00'
System info: host: 'RMFIWAS', ip: '192.168.1.102', os.name: 'Windows 8', os.arch: 'amd64', os.version: '6.2', java.version: '1.7.0_45'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.execute(FirefoxDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:527)
	at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:276)
	at com.example.tests.TestBase.openMainPage(TestBase.java:77)
	at com.example.tests.ContactCreationTests.testNonEmptyContactCreation(ContactCreationTests.java:10)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.remote.SessionNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testNonEmptyContactCreation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.example.tests.ContactCreationTests@226dcb0f]" name="tearDown" is-config="true" duration-ms="0" started-at="2013-10-26T16:14:02Z" finished-at="2013-10-26T16:14:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.example.tests.ContactCreationTests@226dcb0f]" name="tearDown" is-config="true" duration-ms="0" started-at="2013-10-26T16:14:02Z" finished-at="2013-10-26T16:14:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.example.tests.ContactCreationTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
